packages/api/src/controllers/user.ts

```diff
@@ -523,6 +524,66 @@ app.patch(
   }
 );
 
+app.patch(
+  "/:id/disabled",
```

Comment:
Could also be just `PATCH /user/:id` as it's patching the user object

Reply:
Actually, is this even an API? I thought it would be automatically enabled when hitting billing limits, and automatically cleared once payment is made.

Reply:
It's an api similar to :id/suspended to be used by admins